pipeline/runtime/common.go

```diff
@@ -118,6 +118,25 @@ func fetchExportedEnvVars(envFile string, out io.Writer) map[string]string {
 	return env
 }
 
+// Fetches output variables exported by the step.
+func fetchOutputVariablesFromEnvFile(outputFile string, out io.Writer) map[string]string {
```

Comment:
this method is a copy of above one. Can we de-duplicate it?

Reply:
Let me make a common one.